# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1481658678 18000
#      Tue Dec 13 14:51:18 2016 -0500
# Branch stable
# Node ID b3f5fe7e6f756efa97ed09dd4eabae4cb91eb1b3
# Parent  0419dec5d5c1b30aa0c7bb0ce55db1ad91a526f4
gnulib: import fix for link.m4

diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,1 +1,1 @@
-a05c0ede662080761b9b05febaa01698b1ee1474 gnulib
+ee9a87662d22a6af2ae81b637badccdf625144c3 gnulib
